### ISSUE TITLE
SERIOUS · WCAG 2.0 A issues on reCaptcha

### ISSUE THREAD

**Issue by jenfloods** [google/recaptcha#503]
Running Web Accessibility checker and Accessibility Web RAMP tools and it flags the reCaptcha for the following, please resolve ASAP

<html> element must have a lang attribute
SERIOUS · WCAG 2.0 A
Ensures every HTML document has a lang attribute.

Success criteria 3.1.1

Highlight All 1 Elements
Highlight elementiframe[title="reCAPTCHA"],html
<html jstcache="0">
Failure Details
The <html> element does not have a lang attribute

-----------------------------------------------------------

Documents must have <title> element to aid in navigation
SERIOUS · WCAG 2.0 A
Ensures each HTML document contains a non-empty <title> element.

Success criteria 2.4.2

Highlight All 1 Elements
Highlight elementiframe[title="reCAPTCHA"],html
<html jstcache="0">
Failure Details
Document does not have a non-empty <title> element

**Comment by rowan-m** [comment]
Repo is just for PHP client issues. This is about the reCAPTCHA service.
